packages/pwa-kit-react-sdk/src/ssr/server/express.js

```diff
@@ -165,7 +167,11 @@ export const createApp = (options) => {
         // be no use-case for SDK users to set this.
         strictSSL: true,
 
-        enableLegacyRemoteProxying: true
+        enableLegacyRemoteProxying: true,
+
+        // This boolean value indicates whether or not the express
+        // server uses body-parser middleware for POST requests.
+        enableBodyParser: true
```

Comment:
@kevinxh – if our plan is to deprecate this, would we call it `enableLegacyBodyParser` to keep things consistent?

Reply:
Also, would you consider adding a warning similar to how we do it for the other flag?

https://github.com/SalesforceCommerceCloud/pwa-kit/blob/7852d3103c72be098eb8dc6ccfb9eb6a200f795f/packages/pwa-kit-react-sdk/src/ssr/server/express.js#L435-L442

Reply:
@kevinxh I know Vincent gaave you the +1 . But I'd make the above change before merging.